src/main/java/com/dreammaster/scripts/ScriptBiomesOPlenty.java

```diff
@@ -103,9 +103,10 @@ public void loadRecipes() {
                 getModItem(BiomesOPlenty.ID, "misc", 1, 1, missing),
                 getModItem(BiomesOPlenty.ID, "misc", 1, 1, missing),
                 getModItem(BiomesOPlenty.ID, "misc", 1, 1, missing));
-        addShapelessRecipe(
-                getModItem(BiomesOPlenty.ID, "planks", 2, 14, missing),
-                getModItem(BiomesOPlenty.ID, "logs4", 1, 3, missing));
+        /*
```

Comment:
if we not need this just remove it i would say.

Reply:
oh yea was just testing first. forgot to remove it proper at the end. one second